src/helpers/__test__/messageHelper.test.js

```diff
@@ -11,54 +11,72 @@ describe( 'messageHelper', () => {
                 type: 'message'
             };
         } );
-
-        test( 'subtype is "bot_message"', () => {
-            event.subtype = 'bot_message';
-            expect( messageHelper.messageIsFromABot( event ) ).toEqual( true );
-        } );
-
-        test( 'subtype is not "bot_message"', () => {
-            event.subtype = 'not bot_message';
-            expect( messageHelper.messageIsFromABot( event ) ).toEqual( false );
-        } );
-
-        test( 'bot_profile.name is "bot"', () => {
-            event.subtype = 'not bot_message';
-            event.bot_profile = {
-                name: 'bot'
-            };
-
-            expect( messageHelper.messageIsFromABot( event ) ).toEqual( true );
-        } );
-
-        test( 'bot_profile.name is not "bot"', () => {
-            event.subtype = 'not bot_message';
-
-            event.bot_profile = {
-                name: 'not bot'
-            };
-
-            expect( messageHelper.messageIsFromABot( event ) ).toEqual( false );
+        
+        describe( 'subtype', () => {
+            test( '"bot_message"', () => {
```

Comment:
Why are we doing double quotes inside of single quotes here?

Reply:
Because that’s an actual stream that comes from Slack. So I’m quoting it the same way you “what somebody says.